docs/api-reference-1/wordpress-source.md

```diff
@@ -237,18 +274,18 @@ This action fetches all entities related to a `link`, i.e. the pathname of a URL
 All received data are populated in `state.source` and are accessible using the methods explained in the next section.
 
 ```javascript
-actions.source.fetch("/category/nature/");
+actions.source.fetch("/category/nature/").then(dataFetched => ...)
```

Comment:
Actions don't return data. Data is always accessed via the `state`.

That's because Frontity is following the [Flux pattern](https://facebook.github.io/flux/), like Redux.